integration-cli/requirements_test.go

```diff
@@ -99,6 +100,17 @@ func Devicemapper() bool {
 	return strings.HasPrefix(testEnv.DaemonInfo.Driver, "devicemapper")
 }
 
+// containerdSnapshotterEnabled checks if the daemon in the test-environment is
+// configured with containerd-snapshotters enabled.
+func containerdSnapshotterEnabled() bool {
+	for _, v := range testEnv.DaemonInfo.DriverStatus {
+		if v[0] == "driver-type" {
```

Comment:
I was originally thinking of that, but was a bit on the fence as the `DriverStatus` field is not really a formal contract, and adding a `const` for it (if exported) may give the suggestion it is part of the API contract.

I can see value to prevent us (accidentally) renaming the field, and then this test would not check for the right thing (likely the test would fail in that case of course 😂).